components/ui/AddPlantModal.tsx

```
@@ -19,16 +19,18 @@ import { OptimizedIcon } from './OptimizedIcon';
 
 interface AddPlantModalProps {
   visible: boolean;
   onClose: () => void;
   onSuccess: () => void;
 }
-
+// ...existing code...
 export function AddPlantModal({ visible, onClose, onSuccess }: AddPlantModalProps) {
   const { t } = useTranslation('addPlantModal');
   const { height: screenHeight } = useWindowDimensions();
+  const { t } = useI18n();
```

Biome (1.9.4)

[error] Shouldn't redeclare 't'. Consider to delete it or rename it.

't' is defined here:


(lint/suspicious/noRedeclare)